File: core/common/src/main/java/com/shadov/codehellven/common/Check.java
package com.shadov.codehellven.common;

import org.apache.commons.lang3.StringUtils;

import java.util.Arrays;
import java.util.Objects;

public class Check {
	public static boolean noBlanks(String... args) {
		return Arrays.stream(args).noneMatch(StringUtils::isBlank);
	}

	public static boolean anyBlank(String... args) {
		return Arrays.stream(args).anyMatch(StringUtils::isBlank);
	}

	public static boolean isNull(Object object) {
		return Objects.isNull(object);
	}

	public static boolean notNull(Object object) {
		return !Objects.isNull(object);
	}
}

File: core/common/src/main/kotlin/com/shadov/codehellven/common/Check.kt
package com.shadov.codehellven.common

import org.apache.commons.lang3.StringUtils

import java.util.Arrays
import java.util.Objects
import java.util.function.Predicate

object Check {
    fun noBlanks(vararg args: String): Boolean {
        return Arrays.stream(args).noneMatch { StringUtils.isBlank(it) }
    }

    fun anyBlank(vararg args: String): Boolean {
        return Arrays.stream(args).anyMatch { StringUtils.isBlank(it) }
    }

    fun isNull(`object`: Any): Boolean {
        return Objects.isNull(`object`)
    }

    fun notNull(`object`: Any): Boolean {
        return !Objects.isNull(`object`)
    }
}
